src/reducers/vis-state-updaters.js

```diff
@@ -652,6 +652,9 @@ export const addFilterUpdater = (state, action) =>
  */
 export const layerColorUIChangeUpdater = (state, {oldLayer, prop, newConfig}) => {
   const newLayer = oldLayer.updateLayerColorUI(prop, newConfig);
+  if (prop === 'colorRange') {
```

Comment:
this check makes sure we update layer data only when we change the color range property.